src/main/java/de/tum/in/test/api/util/ReflectionTestUtils.java

```diff
@@ -82,6 +83,31 @@ public static Object newInstance(String qualifiedClassName, Object... constructo
 		return newInstance(getClazz(qualifiedClassName), constructorArgs);
 	}
 
+	/**
+	 * Instantiate an object of a class by its qualified name and the constructor
+	 * arguments, if applicable.
+	 * <p>
+	 * This method does not support passing null, passing subclasses of the
+	 * parameter types or invoking constructors with primitive parameters. Use
+	 * {@link #newInstance(Constructor, Object...)} for that.
```

Comment:
The link here should probably point to the private version.

Reply:
Good catch, fixed here and for the other two locations as well.